.nais/application/application-config-dev.yaml

```diff
@@ -32,7 +32,6 @@ spec:
     cpuThresholdPercentage: 75
   resources:
     limits:
-      cpu: "2"
```

Comment:
👍 [NAIS-doc](https://doc.nais.io/workloads/explanations/good-practices/?h=limit+cpu#set-reasonable-resource-requests-and-limits)